nextclade/nextclade.cwl

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+doc: Assign Nextstrain clades to SARS-CoV-2 sequences and provide QC information
+id: nextclade
+label: Nextclade
+
+dct:creator:
+  "@id": "https://orcid.org/0000-0001-6553-5274"
+  foaf:name: Peter van Heusden
+  foaf:mbox: "mailto:pvh@sanbi.ac.za"
+
+requirements:
+  DockerRequirement:
+    dockerPull: neherlab/nextclade:0.8.1-alpine
+
+hints:
+  ResourceRequirement:
+    coresMin: 1
+    ramMin: 512  # 512 MB
+
+inputs:
+  input_fasta:
+    type: File
+    doc: .fasta or .txt file with input sequences
+    format: 
+      - edam:format_1929  # FASTA
+      - edam:format_1964  # plain text format
```

Comment:
```suggestion
      - iana:text/plain
```

And add `iana: https://www.iana.org/assignments/media-types/` to the `$namespaces` section